saas/labs/lab2-create-landing-page/README.md

```diff
@@ -212,10 +212,9 @@ Once you see a form to fill out to purchase your SaaS Offer, you can fill it out
 
 **You are now acting in the role of the customer.**
 
-1. Under the **Publisher signoff** section of the Offer overview page, click teh **Azure portal preview** link.
-
-    You may need to log in to the portal.
-
+1. In the command bar at the top of the Azure portal, search for and select **Marketplace**.
```

Comment:
I wondered about going through PC or through the portal for this. I think I like your method better. I'll keep it.